README.md

```diff
@@ -1,17 +1,72 @@
-# menu
+# Restaurant Menus module
 
+What are your service's inputs and outputs (API Spec)?
 
+`GET /:restaurant_id/restaurant_menus/`
 
-<!-- let menuSchema = new mongoose.Schema({
+**Input**: `restaurant_id` identifies which restaurant to get menus
+
+**Output**: JSON of current restaurant menu(s) data
+
+```
+{
+  “restaurant_id”: <number>,
+  "website": <string>,
+  “menus”: [{
+             “menu_name”: <string>,
```

Comment:
Indenting is a little strange here. Is there a specific reason?

Reply:
Modified!